src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -1986,9 +1957,24 @@ def test_uniq_proxy_declns_invalid_access():
         psy.invokes.invoke_list[0].unique_proxy_declarations(
             ["gh_field"],
             access="invalid_acc")
-    assert ("DynInvoke.unique_proxy_declarations() called with an invalid "
-            "access type. Expected one of {0} but found 'invalid_acc'.".
-            format(valid_access_names) in str(excinfo.value))
+    assert ("Expected one of {0} as a valid access type but found "
+            "'invalid_acc'.".format(valid_access_names) in str(excinfo.value))
+
+
+def test_uniq_proxy_declns_invalid_intrinsic_type():
+    ''' Tests that we raise an error when DynInvoke.unique_proxy_declarations()
+    is called for an invalid intrinsic type. '''
+    from psyclone.dynamo0p3 import MAPPING_DATA_TYPES
```

Comment:
Please move this import to the top of the file.

Reply:
Done.